### ISSUE TITLE
Change the certification to a booleanfield and not a property

### ISSUE THREAD

**Issue by Mortinat** [UrLab/DocHub#281]
The problem is that not all documents tagged "official" are certified, so it would be better to have a specific boolean field.

**Comment by C4ptainCrunch** [comment]
Indeed, it seems a good idea to use a dedicated field and not the fact that the "official" tag.
If someone wants to learn how to write migrations, this could be a nice first issue as we will have to set the field to `True` for every document that has the `official` tag at the moment, then remove the tag.

**Comment by Mortinat** [comment]
Not sure, it's a good idea to do it like that because official just mean it as been given by the uv/teacher. But certified is not mean to be that, the idea would be to have some moderator that determine which is the best synthesis etc

**Comment by C4ptainCrunch** [comment]
See https://github.com/DocHub-ULB/DocHub/pull/296 for a more comprehensive discussion.
